code/modules/urist/modules/shipbattles/weapons.dm

```diff
@@ -282,6 +282,119 @@
 	desc = "It's a big warhead for a big torpedo. Shove it in a torpedo casing and you've got yourself a torpedo." //torpedo
 	icon = 'icons/urist/items/ship_projectiles.dmi'
 	icon_state = "torpedowarhead"
+	var/safety = 1 //Integrated safeties are functioning as intended. This thing's inert.
+	var/is_rigged = 0 //Has this thing been jerry-rigged?
+	var/riggedstate = 0 //What stage of rigging are we in?
+	var/obj/item/device/attached_device //Whatever is attached that'll detonate us.
+
+/obj/item/shipweapons/torpedo_warhead/examine(mob/user)
+	..(user)
```

Comment:
The arguments are automatically passed by calling `..()`. You'll also need to check if `..()` returned false and not display anything in that case. Finally, you'll want to set the return value `.` equal to `..()` for the case some future type also overwrites this proc.